addons/mail/models/models.py

```diff
@@ -31,3 +34,123 @@ def _notify_email_header_dict(self):
         return {
             'X-Odoo-Objects': "%s-%s" % (self._name, self.id),
         }
+
+    @api.model
+    def _notify_admins(self, subject, body, doc=False, **kwargs):
+        """
+            Posts notifications about system failures on Admins channel (recreates the channel if it no longer exists).
+        """
+        ir_model_data = self.env['ir.model.data']
+        model_mail_channel = self.env['mail.channel']
+        odoobot_id = ir_model_data.xmlid_to_res_id("base.partner_root")
+        # Get channel
+        admins_channel = ir_model_data.xmlid_to_object('mail.channel_admins') or model_mail_channel.sudo().search([('name', '=', str('Administrators'))])
+        # Create a new channel if it no longer exists
+        if not admins_channel:
+            admins_channel = model_mail_channel.sudo().with_context(mail_create_nosubscribe=True).create({
```

Comment:
Wouldn't it be better to create this channel in data + prevent the deletion by overriding the unlink ?